page-vodka.php

```diff
@@ -0,0 +1,94 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="">
+    <div <?php post_class('container'); ?> id="post-<?php the_ID(); ?>">
+    
+
+<?php
+if (have_posts()) {
+  while (have_posts()) {
+    the_post();
+    $vodka_video = get_post_meta(get_the_ID(), '_igv_vodka_video_group', true);
```

Comment:
you don't need to run `get_the_ID()` here, it's already in the global scope. you can use `$post->ID`